_content/reference/frequently-asked-meditation-questions_yuttadhammo.md

```diff
@@ -0,0 +1,17 @@
+---
+title: "Frequently Asked Meditation Questions"
+authors:
+  - yuttadhammo
+external_url: "https://sirimangalo.org/frequently-asked-meditation-questions/"
+drive_links:
+  - "https://drive.google.com/file/d/12164Ulh55IwZ1lIPZoQ4uValhaI4M4Pm/view?usp=sharing"
+course: problems
+tags:
+  - sati
+  - vipassana
+pages: 5
```

Comment:
needs a year. It was published in 2013: https://web.archive.org/web/20130812013838/http://www.sirimangalo.org/teachings/frequently-asked-meditation-questions